services/quests/src/playtest/PlaytestCrawler.tsx

```diff
@@ -89,9 +89,9 @@ export class PlaytestCrawler extends StatsCrawler {
     }
   }
 
-  protected onWarnings(q: StatsCrawlEntry, warnings: Error[], line: number) {
```

Comment:
This change seems a little risky out of context, since it makes it look like we're turning all warnings into errors. But, I trust you know what you're doing :)

Reply:
It's only used for reporting op errors, and it doesn't actually affect the app's behavior in handling them... should be fine to change it.

I also developed a "playtest runner" that runs the default search, fetches the XML, and runs the playtester on all of them concurrently when you visit `quests.expeditiongame.com/runner.html`... PTAL.

Reply:
Runner file LGTM, curious how the CI issue gets resolved